src/polus/plugins/__main__.py

```diff
@@ -1,3 +1,9 @@
+"""
+polus-plugins main.py.
+
+main.py
+"""
+
 import click
```

Comment:
We should convert this over to `typer`. We should also see if we can't auto generate typer commands. But, not for this PR.

Reply:
Removed this file